automation/taskcluster/lib/gradle.py

```diff
@@ -0,0 +1,46 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import print_function
+import json
+import subprocess
+
+
+def get_build_variants():
+    print("Fetching build variants from gradle")
+    output = _run_gradle_process('printBuildVariants')
+    content = _extract_content_from_command_output(output, prefix='variants: ')
```

Comment:
It might be worth experimenting with merging `_run_gradle_process` with `_extract_content_...` to see if it reads nicer :man_shrugging:

Reply:
That was my first approach, but I find nicer to understand a function that takes a raw string and extract the valuable data out of it.